lib/doh.c

```diff
@@ -42,9 +42,17 @@
 #include "curl_printf.h"
 #include "curl_memory.h"
 #include "memdebug.h"
+#ifdef USE_HTTPSRR
+# include "escape.h"
+#endif
```

Comment:
No need to ifdef this for just an include. Minimizing the amount of ifdefs makes the code easier on the eye.

Reply:
done